docs/config/index.md

```diff
@@ -55,7 +55,7 @@ Vite also directly supports TS config files. You can use `vite.config.ts` with t
 If the config needs to conditional determine options based on the command (`serve` or `build`) or the [mode](/guide/env-and-mode) being used, it can export a function instead:
 
 ```js
-export default ({ command, mode }) => {
+export default defineConfig({ command, mode }) {
```

Comment:
I agree that we should probably benefit from making `defineConfig` more prominent. But I don't know if we should change almost all instances, maybe a note somewhere in the guide could work out? I let others decide on this one.

Missing a `=>` here:
```suggestion
export default defineConfig({ command, mode }) => {
```

Reply:
Also missed the braces here ^^' thx for the catch